app/views/records/edit_fields/_creator_personal.html.erb

```diff
@@ -1,15 +1,16 @@
 <div data-toggleable-group="<%= type %>">
 	<% creator_inst_rel_options = HykuAddons::InstitutionalRelationshipService.new(model: f.object.model.class).select_active_options %>
+	<% creator_role_options = HykuAddons::RoleService.new(model: f.object.model.class).select_active_options %>
 	<% personal_fields = [
 		{ field_type: :text, field_slug: :creator_family_name, field_args: { data: { required: f.object.required?(:creator), required_group: :creator_name, on_blur: :toggle_required_group } } },
 		{ field_type: :text, field_slug: :creator_given_name, field_args: { data: { required: f.object.required?(:creator), required_group: :creator_name, on_blur: :toggle_required_group } } },
 		{ field_type: :text, field_slug: :creator_orcid },
 		{ field_type: :select, field_slug: :creator_institutional_relationship, select_options: creator_inst_rel_options, field_args: { multiple: true } }] %>
 	<% personal_fields = add_pacific_creator_personal_fields(personal_fields) if template.include? "pacific" %>
```

Comment:
The fact that we have 3 clients and all these require a specific method here, is a pretty big code smell that we need to think of a better way to do this, so that in the future, if we have a thousand customers, we dont have a thouand methods here.  

`add_pacific_creator_personal_fields`, `add_uva_creator_personal_fields`, `add_redlands_creator_personal_fields`

Reply:
@cjcolvar tagging you in here too. We need a different approach.